scripts/clowder/quay.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Usage: cd <svc dir>; export QUAY_ROOT=...; ./quay.sh <svc>
+# Example: ./quay.sh sources-api
+
+if [[ -z $QUAY_ROOT ]]; then
+  echo "Please export QUAY_ROOT=\"quay.io/<username>\" first"
+  exit
+fi
+
+svc=$1
+image_tag=`cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 7 | head -n 1`
+
+# BUILDAH_LAYERS=false podman build -t ... # don't use cached layers
+
+podman build -t ${svc}:${image_tag} .
```

Comment:
@lindgrenj6 some of these commands requires manual select of image:latest from 
- registry.fedoraproject.org
- docker.io
- registry.access.redhat...
- ...
which are not present and obviously are not needed.
Do you know how to skip it?

Reply:
just throw a `quay.io/` in front of it, e.g. `quay.io/${svc}:${image_tag}`.

podman requires fully-qualified images now, instead of defaulting to docker.io

Reply:
@lindgrenj6 it didn't help: the output looks like this:
```
STEP 15: CMD ["run_scheduler"]
STEP 16: COMMIT quay.io/topological-inventory-scheduler:66RP59E
--> 851f02ae29b
851f02ae29b89e85b77145e1c663a6dddeb52fbbd9c115e120a896fc0b3fd243
? Please select an image: 
  ▸ registry.fedoraproject.org/topological-inventory-scheduler:latest
    registry.access.redhat.com/topological-inventory-scheduler:latest
    registry.centos.org/topological-inventory-scheduler:latest
    docker.io/library/topological-inventory-scheduler:latest
```

Reply:
Oh, I forgot to mention we need to include the username, e.g. `quay.io/${QUAY_USER}/...`

It looks like `$QUAY_ROOT` would work.